db/changelog/v-1/11-changeset-test-agent-insert-data.xml

```diff
@@ -20,6 +20,7 @@
             <column name="passed_tests" value="0" />
             <column name="failed_tests" value="0" />
             <column name="skipped_tests" value="0" />
+            <column name="sdk" value="ubuntu:latest" />
```

Comment:
Let's call it `default` or something similar, having image name in sdk column doesn't make much sense for me

Reply:
I can change it to `default` but it will complicate another logic in agent to parse string into `sdk` and then again into string and also, maybe we won't make it default - in fronedn we can alert user, if he doesn't select any sdk?

Reply:
No, some tools don't require runtime, e.g. save-cli is a native binary, which will run on almost any environment. So choosing default runtime without any specific sdk is a valid choice